platform/src/server/http/__tests__/http_config.test.ts

```diff
@@ -38,3 +38,119 @@ test('throws is basepath appends a slash', () => {
   };
   expect(() => httpSchema.validate(obj)).toThrowErrorMatchingSnapshot();
 });
+
+test('throws is basepath is not specified, but rewriteBasePath is set', () => {
```

Comment:
whoops, sorry, ignore the comments on the generated snapshot.
**typo:** first is should be if

Reply:
but I think to get rid of the typos in the snapshot, they need to be regenerated. can you make sure there are no other places where we say "throws is X is"?

Reply:
Sure, looks like it's the only place with `throws is`

Reply:
:heavy_check_mark: